pkg/config/bdb-config.yaml

```diff
@@ -0,0 +1,12 @@
+---
+- name: "bdb-node-01" # Instance name
+  box:
+    name: "ubuntu/xenial64" # Box name
+  ram: "2048"
+  vcpus: "2"
+  setup_type: "quickstart" # Currently, only quickstart is supported.
+  deploy_docker: true # [true, false]
+  network:
+    ip: "10.20.30.50"
+    type: "private_network" # Currently, only private network is supported.
```

Comment:
Comment about `private_network` above. Cross referencing here for clarity.

Reply:
I have added support for `public_network` as well. So this will not be an issue anymore.